test/unit/deprecated_references_test.rb

```diff
@@ -86,20 +89,33 @@ class DeprecatedReferencesTest < Minitest::Test
       deprecated_references = DeprecatedReferences.new(package, "test/fixtures/deprecated_references.yml")
       deprecated_references.add_entries(first_violated_reference, Packwerk::ViolationType::Privacy)
       deprecated_references.add_entries(second_violated_reference, Packwerk::ViolationType::Privacy)
-      assert deprecated_references.stale_violations?
+      assert deprecated_references.stale_violations?(Set.new([
+        "orders/app/jobs/orders/sweepers/purge_old_document_rows_task.rb",
+        "orders/app/models/orders/services/adjustment_engine.rb",
+      ]))
+    end
+
+    test "#stale_violations? returns false if violations in deprecated_references.yml exist but are not found when checking for violations" do
+      package = Package.new(name: "buyers", config: { "enforce_dependencies" => true })
+      deprecated_references = DeprecatedReferences.new(package, "test/fixtures/deprecated_references.yml")
+      assert deprecated_references.stale_violations?(
```

Comment:
This assertion doesn't match the description of the test

Reply:
Thanks, fixed!